up-core-api/uprotocol/uri.proto

```diff
@@ -54,46 +54,30 @@ message UUri {
 // An Authority represents the deployment location of a specific Software Entity.
 // Authority can be represented in either a name (i.e example.com), ip address (205.236.147.1)
 // or an ID (i.e. VIN, SHA 128, or any other identifier that is less than 255 bytes.
-// *NOTE:* When Authority is empty (neither name, ip, or id are set) the authority
-// is local.
 message UAuthority {
-  optional string name = 1; // Domain & device name as a string
-  oneof number {
-    bytes ip = 2; 	// IPv4 or IPv6 Address in byte format
-    bytes id = 3;		// Unique ID for the device, could be a VIN, SHA 128, or any other identifier
-                    // *NOTE:* MAX length is 255 bytes
-  }
+  string name = 1; // Domain & device name as a string
```

Comment:
FMPOV this would need to remain `optional` in order to consistently support matching _any_ value across UUri's components.

Alternatively, we could think about defining explicit wildcard characters/values ...

Reply:
We discussed this and we are using explicit value definitions so there is no optional